packages/expo-modules-core/android/src/main/cpp/types/CppType.h

```diff
@@ -33,5 +33,6 @@ enum CppType {
   VALUE_OR_UNDEFINED = 1 << 21,
   JS_ARRAY_BUFFER = 1 << 22,
   NATIVE_ARRAY_BUFFER = 1 << 23,
+  WORKLET = 1 << 24,
```

Comment:
TBH, I didn't expect that we would handle so many types. We're getting closer and closer to the int limit 😅